### ISSUE TITLE
Danknull rendering crash

### ISSUE THREAD

**Issue by Rynkyden** [sp614x/optifine#3562]
https://paste.dimdev.org/itohapeqeq.mccrash

**Comment by realjackasterisk** [comment]
So this only occurs in multiplayer?

**Comment by Rynkyden** [comment]
No. Using the modpack i was able to recreate in single player. I think it
has something to do with jei. I didnt have time to test with more mods.  I
created a profile with just optifine and danknull.  And the crash did not
occur.
